test/GPv2Settlement.test.ts

```diff
@@ -1068,38 +1066,18 @@ describe("GPv2Settlement", () => {
       };
 
       await expect(
-        settlement.executeInteractionTest(invalidInteraction),
+        settlement.executeInteractionsTest([invalidInteraction]),
       ).to.be.revertedWith("GPv2: forbidden interaction");
     });
 
-    it("should revert when interaction reverts", async () => {
-      const reverter = await waffle.deployMockContract(deployer, [
-        "function alwaysReverts()",
-      ]);
-      await reverter.mock.alwaysReverts.revertsWithReason("test error");
-      const revertingCallData = reverter.interface.encodeFunctionData(
-        "alwaysReverts",
-      );
-      const failingInteraction: Interaction = {
-        target: reverter.address,
-        callData: revertingCallData,
-        value: 0,
-      };
-
-      // TODO - update this error with concatenated version "GPv2 Interaction: test error"
```

Comment:
Is this no longer needed? I guess I can close the issue #240

Reply:
Its still there, just moved to GPv2Interaction.test.ts:
https://github.com/gnosis/gp-v2-contracts/blob/50ad7bef926148b96fc8696da30ef7bc04172edd/test/GPv2Interaction.test.ts#L33

Reply:
Dang. I was hoping we wouldn't need to content error strings in solidity